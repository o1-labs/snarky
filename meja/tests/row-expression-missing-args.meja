let x : [< `A(int) | `B | `C > `A] = `A;
