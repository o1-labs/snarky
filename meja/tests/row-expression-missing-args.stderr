File "tests/row-expression-missing-args.meja", line 1, characters 37-39:
Error: Incompatible types ([<`A(int) | `B | `C > `A] as 'a) and
       ([>`A] as 'b):
       Cannot unify ([<`A(int) | `B | `C > `A] as 'a) and ([>`A] as 'b):
       the constructor `A has different arities.
